### PR TITLE
feat/352-implement-SumSub-SDK

### PR DESCRIPTION
**Pull request by ciripel**

### CODE REVIEW

File: lib/view/routers/authenticated_routes/sumsub_route.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:go_router/go_router.dart';
+import 'package:simplio_app/view/mixins/page_builder_mixin.dart';
+import 'package:simplio_app/view/helpers/application_route.dart';
+import 'package:simplio_app/view/screens/authenticated_screens/sumsub_screen.dart';
+
+class SumSubRoute extends ApplicationRoute with PageBuilderMixin {
```
Comment (storkandstars) [inline lib/view/routers/authenticated_routes/sumsub_route.dart:6]: We should not use the SumSub name as the kyc provider can change anytime

Comment (storkandstars) [reply lib/view/routers/authenticated_routes/sumsub_route.dart:6]: What is the reason to have a specific route for kyc?

File: lib/view/screens/authenticated_screens/sumsub_screen.dart [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  Widget build(BuildContext context) {
+    return WillPopScope(
+      onWillPop: () async => false,
+      child: SioScaffold(
+        resizeToAvoidBottomInset: false,
+        body: SafeArea(
+          child: Stack(
+            fit: StackFit.expand,
+            children: [
+              InAppWebView(
```
Comment (storkandstars) [inline lib/view/screens/authenticated_screens/sumsub_screen.dart:74]: Why do we use WebView here?

File: lib/view/routers/authenticated_routes/sumsub_route.dart (hunk 1 above)
Comment (ciripel) [reply lib/view/routers/authenticated_routes/sumsub_route.dart:6]: It is not.... Delete them... I thought we will use it in webview so I created a webview and route... They should be deleted. They are not needed

**Pull request merged**